Chirich/Lab1.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
_relates to the files above_
Please. Use. Gitignore. I see you found the way to see other people's tasks :). So, please read my comments after reading other people's code, where I say a lot about using gitignore and why it's important.